ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -579,33 +468,366 @@ def click_actions_control_then_start(self):
+        self.driver.execute_script("arguments[0].click();", el)
+        logger.info("Opened Volume name dropdown")
+
+        first_centos_loc = self.vm_loc["add_volume_first_centos_stream_option"]
+        wait_for_element_to_be_visible(locator=first_centos_loc, timeout=20)
+        first_centos_el = self.driver.find_element(
+            first_centos_loc[1], first_centos_loc[0]
+        )
+        selected_vol_text = first_centos_el.text.strip()
+        self.driver.execute_script("arguments[0].click();", first_centos_el)
+        logger.info(
+            f"Selected Volume name: '{selected_vol_text}' (first centos-stream)"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)